README.md

```diff
@@ -2,55 +2,31 @@
 
 Hello there!
 
-At Seenons, we are building a platform to support a circular economy. This encompasses collection, transportation and
-re-purposing of different waste streams. A strong technical backbone, design and usability are key to the success of the
-platform!
+At Seenons, we are building a platform to support a circular economy. This encompasses collection, transportation and re-purposing of different waste streams and our assignment is a subset of what we do.
 
 ## The assignment
 
 ---
 
-We have divided the assignment into what we called "gates", these gates are references to engineering skills that we are looking for in a future colleague.
-Some of them are required, some of them give you bonus points. You can read more about it below.
+The task is to at least complete two use cases and their corresponding specs.
 
-    The task is to at least complete two services
-    - Retrieve availability of service providers for a given location and date.
-    - Register the collection of a waste stream to be performed by a service provider.
+- Retrieve availability of service providers at a given location, waste stream and date.
+  - see: <https://github.com/seenons/seenons-be-assignment/blob/main/src/usecases/availability/find_availability_at.ts>
+- Register a waste stream pickup to be performed by a service provider.
+  - see: <https://github.com/seenons/seenons-be-assignment/blob/main/src/usecases/pickup/register_waste_pickup.ts>
 
-We purposely made this project framework-agnostic, and it only contains the bare minimum to perform the necessary use cases.
+You can use the provided code as a starting point or rebuild it however you see fit, including the domain representations in case you do not agree with its contents.
 
-## The output
+## The delivery
 
 ---
 
-    Code changes that reflect the implementation of at least the two required use cases
-    In case you make changes to the domain model (as an opportunity), include a diagram that explains it.
-    Ensure that test coverage is adequate and that it matches what you would deliver as a product engineer.
-    Create a new readme file that explains your take on the assignment, your thought process, and any other information that you believe is relevant for the review.
+- Create a copy of this repository, do not fork it.
```

Comment:
Maybe be specific about this, since you mention private in the last bullet point
```suggestion
- Create a **private** copy of this repository. Do not fork it.
```